decred/decred/dcr/account.py

```diff
@@ -1383,7 +1400,7 @@ def nextAddr():
             try:
                 addr = branchKey.deriveChildAddress(idx, self.net)
             except crypto.CrazyKeyError:
-                addr = CrazyAddress
+                return CrazyAddress
```

Comment:
It does not seem useful to add a CrazyAddress to the data, it will be discarded by the caller anyway.

Reply:
Yep. It's only purpose is index preservation in various contexts.

Reply:
Oops, reverting.